packages/@o3r/core/schematics/rule-factories/monorepo/index.ts

```diff
@@ -0,0 +1,36 @@
+import type { Rule, SchematicContext, Tree } from '@angular-devkit/schematics';
+import { SchematicsException } from '@angular-devkit/schematics';
+import * as commentJson from 'comment-json';
```

Comment:
we should try to avoid using comment-json in schematics @kpanot has open a PR to remove it